rule.proto

```diff
@@ -0,0 +1,95 @@
+syntax = "proto2";
+package dv360_excel_plugin;
+
+/* A message that represents a scheduled rule for manipulating DV360 entities.
```

Comment:
If comment is only one line, can we make it like this?
/* A message that represents a scheduled rule for manipulating DV360 entities. */

Also, if the comment is inside a message, maybe we can just use //. In fact in dv3 public api proto files, it only use "//"
e.g. 
// required
// The name of the rule, set by the user.

https://developers.google.com/protocol-buffers/docs/proto3#adding_comments

Reply:
Ok, sure!